Lib/fontParts/base/base.py

```diff
@@ -364,26 +577,66 @@ def get(self, key, default=None):
             value = self.valueNormalizer.__func__(value)
         return value
 
-    def _get(self, key, default=None):
-        """
-        Subclasses may override this method.
+    def _get(self, key: Any, default: Optional[Any]) -> Any:
+        """Get the value for a given key in the native object.
+
+        This is the environment implementation of :meth:`BaseDict.get`.
+
+        :param key: The key to look up. If a :cvar:`BaseDict.keyNormalizer` is
+            set, it will have been applied to the given key.
+        :param default: The default value to return if the key is not found.
+        :return: The value associated with the given key, or the default value
+            if the key is not found. If a :cvar:`BaseDict.valueNormalizer` is set,
+            it will have been applied to the returned value.
```

Comment:
better to say `it will be applied in the calling method to the returned value`

Reply:
Changed.